.github/dependabot.yml

```diff
@@ -0,0 +1,5 @@
+# Dependabot config docs
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+version: 2
+updates:
+  target-branch: 'dependabot-updates-24-03'
```

Comment:
my guess is this should be changed to `master`

Reply:
This will be `master` on final commit. So it will get merged then look at `master`.